### ISSUE TITLE
Dashboards: Unable to disable angularDeprecationUI feature using environment variables

### ISSUE THREAD

**Issue by erlfos** [grafana/grafana#88364]
### What happened?

Setting the environment variable 
`GF_FEATURE_TOGGLES_ANGULARDEPRECATIONUI=False`
does not prevent the Angular deprecation warning popup.
It could be that this the wrong syntax, but it is not well documented how to use environment variables to disable feature toggles.

### What did you expect to happen?

No Angular deprecation warning is displayed when I open a dashboard with a deprecated panel.

### Did this work before?

No, I have not used the GF_FEATURE_TOGGLES env vars before Grafana v11.0, but I use env vars to do all the other Grafana configuration.

### How do we reproduce it?

1. Set env var GF_FEATURE_TOGGLES_ANGULARDEPRECATIONUI=False
2. Start Grafana
3. Create a dashboard with a panel visualization using Angular, for example FlowCharting
4. Observe warning

### Is the bug inside a dashboard panel?

No

### Environment (with versions)?

Grafana: 11.0
Docker image: grafana/grafana-oss:11.0.0

### Grafana platform?

Docker

```[tasklist]
### Tasks
```

**Comment by tonypowa** [comment]
Hi @erlfos 

Thanks for creating this issue

According to this [deprecation notice](https://grafana.com/docs/grafana/latest/developers/angular_deprecation/#when-will-angular-plugins-stop-working), the env variable that you want to switch off is `angular_support_enabled`. In a docker compose file, that would be `GF_SECURITY_ANGULAR_SUPPORT_ENABLED=false`. Note: from Grafana v11, that should default to `false`. 

If you still need to play with the feature flag, 

Have you tried using a different casing for the environment variable, (e.g.,  `false` vs False)? 

>GF_FEATURE_TOGGLES_ANGULARDEPRECATIONUI=False

**Comment by erlfos** [comment]
Hi @tonypowa 
I am sorry if I wasn't being clear. I have set this environment variable to True,
`GF_SECURITY_ANGULAR_SUPPORT_ENABLED=true`, 
i.e I want to still be able to use Angular based panels in Grafana v11. That works fine.
What I don't want is to have the annoying Angular depreciation warning pop-up all the time. I should be able to disable this by disabling the feature toggle called "angularDeprecationUI" which is enabled by default according to these docs: https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/
However I have not found a way to do this using environment variables. I can _enable_ feature toggles by using (just as an example): GF_FEATURE_TOGGLES_ENABLE=newNavigation
but I am not able to _disable_ feature toggles, which is what I want to do. 
So my question is basically, how can I turn this INI file snippet into an environment variable:
```
[feature_toggles]
angularDeprecationUI = false
```

**Comment by tonypowa** [comment]
Thanks for your reply, and no worries !

I don't know the answer, so I will forward this issue to the team 😉

Note for the reviewer: please reroute if the issue does not belong to your squad

**Comment by Clarity-89** [comment]
@grafana/plugins-platform-frontend looks like this one is for you.

**Comment by thedoc31** [comment]
Can confirm that the environment variable feature toggle does not turn off the Angular warning. The `settings.ini` method, however, does. For Dockerized grafana, that means we'd have to mount in a `settings.ini` file in addition to following convention by using environment variables passed in at container start time.

Example as set in my AWS userdata script

```
... prior commands ...
"docker run -d --name grafana -p 3000:3000",
" -e \"HOSTNAME=the_host.yourhost.com\"",
" -e \"GF_INSTALL_PLUGINS=blackmirror1-singlestat-math-panel,farski-blendstat-panel,fzakaria-simple-annotations-datasource,grafana-piechart-panel,jdbranham-diagram-panel,ryantxu-ajax-panel,yesoreyeram-boomtable-panel\"",
" -e \"GF_FEATURE_TOGGLES_ENABLE=newNavigation\"",
" -e \"GF_FEATURE_TOGGLES_ANGULARDEPRECATIONUI=false\"",
... subsequent commands ...
```

Based on code at [1], this appears to be the proper variable to set.

I'd also like to comment that it's unclear based on documentation [2] if setting the `GF_FEATURE_TOGGLES_ENABLE` variable is also required to get any feature toggle to activate when using ENV vars.

[1]
https://github.com/grafana/grafana/blob/main/pkg/services/featuremgmt/registry.go#L575-L582
[2]
https://github.com/grafana/grafana/blob/main/docs/sources/setup-grafana/configure-grafana/_index.md?plain=1#L91

**Comment by Slach** [comment]
@thedoc31 I hit the same issues

Could you explain what exaclty means `settings.ini` method how to partially override `/etc/grafana/grafana.ini`?

**Comment by Slach** [comment]
I need to disable some feature toggles, how to do it in `docker-compose.yaml`

**Comment by erlfos** [comment]
> > I need to disable some feature toggles, how to do it in `docker-compose.yaml`
> 
> You should be able to do it via [environment variables](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables) that you pass to Grafana in the docker-compose file. You can see the values [here](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#feature_toggles).
> 
> It should be something like: `GF_FEATURE_TOGGLES_<FEATURE_TOGGLE_NAME>: false`

That is what this issue is about. We are not able to disable feature toggles via environment vars before this issue is closed.

**Comment by mckn** [comment]
I had a look at the code and it seems like we don't use the `GF_FEATURE_TOGGLES_*` settings when creating the feature toggle settings. So I will send this issue to the proper team so they can fix that.

**Comment by mckn** [comment]
Looks like the docs was updated in this [PR](https://github.com/grafana/grafana/pull/69438).

**Comment by danielcb** [comment]
So, at the moment it's not possible to disable features with environment variables?

**Comment by mckn** [comment]
> So, at the moment it's not possible to disable features with environment variables?

No, but I think this [PR](https://github.com/grafana/grafana/pull/90178) will fix that.

**Comment by thedoc31** [comment]
Should this still be marked as Stale? @mckn Any idea when the PR can be merged?

**Comment by mckn** [comment]
Hmm, I'm not sure. @xnyo any updates on the open PR?
